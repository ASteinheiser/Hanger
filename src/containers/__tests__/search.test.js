import React from 'react';

import Search from '../search.js';

describe('Search tests ->', () => {

  describe('Basic tests ->', () => {

    it('renders a search page', () => {
      const component = shallow(
        <Search />
      );
      expect(component).toMatchSnapshot();
    });
  });
});
