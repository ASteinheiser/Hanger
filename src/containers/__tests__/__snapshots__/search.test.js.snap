// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search tests -> Basic tests -> renders a search page 1`] = `
<Styled(Component)>
  <SearchBar
    onChange={[Function]}
    value=""
  />
  <Styled(ScrollView)
    color="#ffffff"
  >
    <HeaderText
      small={true}
      text="Explore Events, Gigs, etc!"
    />
    <FeedPost
      image="https://www.zonamedspa.com/wp-content/uploads/PHXFW.16-Silver-Flyer.jpg"
      title="Phoenix Fashion Week 2018"
    />
    <FeedPost
      image="https://i.pinimg.com/736x/d6/7a/3b/d67a3bac1c2deb60e02dd8373db472a0--behind-the-scenes-shooting.jpg"
      title="Photographer and 2 Models needed for Paid Photoshoot"
    />
  </Styled(ScrollView)>
</Styled(Component)>
`;
